.github/workflows/regression-tests.yml

```
@@ -0,0 +1,91 @@
+name: REGRESSION TESTER - AGGLAYER
+on:
+  workflow_dispatch:
+    inputs:
+      zkevm_agglayer_commit_id:
+        description: '0xPolygon/agglayer (commit id)'
+        required: true
+      bake_time:
+        description: 'bake time (minutes)'
+        required: true
```

I would set a default value for the bake time, what do you think?

```suggestion
        required: false
        default: 30
```

---

good call, adding